tests/integration/test_data/npm_smoketest_lockfile2/bom.json

```diff
@@ -0,0 +1,387 @@
+{
+  "bomFormat": "CycloneDX",
+  "components": [
+    {
+      "name": "bar",
+      "version": "1.0.0",
+      "type": "library"
+    },
+    {
+      "name": "baz",
+      "version": "1.0.0",
+      "type": "library"
+    },
+    {
+      "name": "eggs-packages/eggs",
+      "version": "1.0.0",
+      "type": "library"
+    },
+    {
+      "name": "foo",
+      "version": "1.0.0",
+      "type": "library"
+    },
+    {
+      "name": "node_modules/abbrev",
```

Comment:
In lockfile version1 and 3 it's easy. For version1 I created SBOMs from dependencies and in the version from packages. Do we want to report always the same version/structure of SBOMs json for all lockfile versions?? Should I tweak it a little bit for the other one?? What about version 2. I based that one from packages as version 3 (latest version the greatest).

Plus should I remove `node_modules/` prefix here

Reply:
`packages` and `dependencies` in `package-lock.json` do not provide always same info. For example for local dependencies packages provide also versions so the output of `npm_smoketest_lockfile2/bom.json` and `npm_smoketest_lockfile1/bom.json` is slightly different. I hope it's fine.

Reply:
Oh, interesting. We can report the version when it's available :+1: